src/topology/algebra/group.lean

```diff
@@ -940,6 +940,28 @@ local attribute [instance] topological_group.regular_space
 @[to_additive]
 lemma topological_group.t2_space [t1_space G] : t2_space G := regular_space.t2_space G
 
+variables {G} (S : subgroup G) [subgroup.normal S] [hS : is_closed (S : set G)]
+
+include hS
+
+@[to_additive]
+instance subgroup.t1_quotient_of_is_closed :
+  t1_space (G ⧸ S) :=
+begin
```

Comment:
Typeclasses are automatically included as needed so I think it is slightly nicer to set this up as:
```suggestion
variables {G} (S : subgroup G) [subgroup.normal S] [is_closed (S : set G)]

@[to_additive]
instance subgroup.t1_quotient_of_is_closed :
  t1_space (G ⧸ S) :=
begin
  have hS : is_closed (S : set G) := by apply_instance,
```